azkaban-common/src/test/resources/flow_filter2.txt

```diff
@@ -0,0 +1,4 @@
+proj1:flow1
+proj2:flow2
```

Comment:
did we consider using a well-defined format like json/xml? It could be simple in this case but I am wondering about the custom code to read and parse this correctly.

Reply:
This is a simple use case. I would use JSON for a complex structure.